multi-node.sh

```diff
@@ -110,6 +110,15 @@ for IP in ${INSTANCE_IPS[@]}; do
 done
 wait
 
+for IP in ${INSTANCE_IPS[@]}; do
+    ssh -o ConnectTimeout=30 -o StrictHostKeyChecking=no -T -i ~/${slave_keypair} ${ami[1]}@${IP} \
+        "sudo reboot" 2>&1 | tr \\r \\n | sed 's/\(.*\)/'$1' \1/'
```

Comment:
$1 should be ${IP}

Reply:
I did see $1 though, there is ami[1] which is user name?

Reply:
I see it now, fixed in updated PR